src/Conf.ml

```diff
@@ -1933,39 +1923,33 @@ let build_config ~file =
     let f = function `Ocamlformat _ -> false | `Ocp_indent _ -> true in
     List.for_all files ~f
   in
+  let quiet = conf.quiet in
   if no_ocamlformat_files && not enable_outside_detected_project then (
     ( if not conf.quiet then
-      let reason =
-        match project_root with
-        | Some root ->
+      let why =
+        Option.value_map project_root ~default:"no project root was found"
```

Comment:
I think this is adding a lot of complexity and is only a single line shorter

Reply:
done